src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -153,6 +170,26 @@ class LovelacePanel extends LitElement {
     );
   }
 
+  private _lovelaceChanged() {
+    if (this._saving) {
+      this._saving = false;
+    } else {
+      if (!this._changedToast) {
+        this._changedToast = document.createElement("hui-actionable-toast");
```

Comment:
Let's not add more logic to this file. I was hoping you would update our existing notification toast event with actions.

Reply:
I like that idea, the problem I have with it is that the notification toast is a shared toast, meaning that if another notification comes in the toast will be gone? And so will the possibility for refreshing the UI... I think we want this toast to be persistent?

Reply:
Ohw, I now see that the `paper-toast` only allows 1 toast on the screen anyway...

Reply:
Maybe it should be a banner? https://material.io/design/components/banners.html

Reply:
I like the idea of a banner.